README.md

```diff
@@ -1,2 +1,24 @@
-# material-components-web-catalog
-Catalog of Material Components for Web (MDC Web)
+# Material Components for the Web Catalog
+
+This is the [catalog of demos](https://material-components.github.io/material-components-web-catalog/) for Material Components for Web (MDC Web).
+
+## About
+
+[Material Components for Web (MDC Web)](https://github.com/material-components/material-components-web) help developers execute [Material Design](https://www.material.io).
+Developed by a core team of engineers and UX designers at Google, these components enable a reliable development workflow to build beautiful and functional web projects.
+
+## Development
+
+To start a local server of the catalog, run
+```
+npm start
+```
+Then point your browser to http://localhost:3000/.
+
+## Deployment
+
+To deploy the catalog to GitHub pages, run
+```
+npm run deploy
```

Comment:
Is this for everyone? Should we allow the public to be deploying the demo?

Reply:
@lynnjepsen confirmed having it here is fine

Reply:
Cool! Looks good to me then